cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -181,6 +182,12 @@ newtype PrivKeyFile
   = PrivKeyFile FilePath
   deriving (Eq, Show)
 
+newtype TxBodyFile = TxBodyFile FilePath
+  deriving (Eq, Show)
```

Comment:
@erikd should these two have `deriving newtype` ?

Reply:
Yeah probably, but not a big issue. Can be fixed in the next PR set.